extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.css

```diff
@@ -14,6 +14,10 @@
  * limitations under the License.
  */
 
+ .amp-carousel-button {
```

Comment:
Warning: this will hide all carousel buttons on the page when `amp-lightbox-gallery.js` loads, which I don't think is what you want.

nit: fix indentation, there's one extra space before the selector and before the closing brace.

Reply:
This is actually intentional. `amp-carousel-button` class applies to the carousel's native next and prev buttons. They are always visible on desktop, but we want to replace these with custom lightbox controls that we can show / hide. For example, we always want to hide them when text description overflow is on.

Reply:
Native `<amp-carousel>` doesn't have buttons besides `next` and `prev`.

Reply:
Yes but carousels outside the lightbox should have their normal arrows, right? This will break those.

Reply:
OH yes. You are right. They should be descendants of lightbox.